server/src/main/java/com/linecorp/centraldogma/server/internal/api/HttpApiUtil.java

```diff
@@ -160,6 +161,18 @@ public static HttpResponse newResponse(RequestContext ctx, HttpStatus status,
 
     private static HttpResponse newResponse0(RequestContext ctx, HttpStatus status,
                                              @Nullable Throwable cause, @Nullable String message) {
+        // TODO(trustin): Use HttpStatus.isContentEmpty().
+        //                https://github.com/line/armeria/pull/2058
+        checkArgument(!HttpStatusClass.INFORMATIONAL.contains(status.code()),
+                      "invalid status code. status: %d", status.code());
+        switch (status.code()) {
+            case /* NO_CONTENT */ 204:
+            case /* RESET_CONTENT */ 205:
+            case /* NOT_MODIFIED */ 304:
+                throw new IllegalArgumentException("no-content status can not create a response. status: " +
```

Comment:
How about:

    "Can't create a response with content for: " + status

..?

Reply:
Thank you for review. Updated in eee6241.